src/ws/login.html

```diff
@@ -35,6 +35,10 @@ <h4 id="login-error-message" class="pf-c-alert__title"></h4>
               <h4 id="login-info-message" class="pf-c-alert__title"></h4>
             </div>
 
+            <div id="info-group" class="alert alert-info" hidden>
+              <span id="login-info-message"></span>
```

Comment:
This duplicates the ID with the already existing `<h4>` from above.

Reply:
Right, the other commit should not exist. This was just wrong rebase.